sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -500,10 +519,22 @@ public partial class IndexingParameters
     {
         public IndexingParameters() { }
         public int? BatchSize { get { throw null; } set { } }
-        public System.Collections.Generic.IDictionary<string, object> Configuration { get { throw null; } set { } }
+        public System.Collections.Generic.IDictionary<string, object> Configuration { get { throw null; } }
         public int? MaxFailedItems { get { throw null; } set { } }
         public int? MaxFailedItemsPerBatch { get { throw null; } set { } }
     }
+    public static partial class IndexingParametersExtensions
+    {
+        public static Azure.Search.Documents.Indexes.Models.IndexingParameters ExcludeFileNameExtensions(this Azure.Search.Documents.Indexes.Models.IndexingParameters parameters, params string[] extensions) { throw null; }
+        public static Azure.Search.Documents.Indexes.Models.IndexingParameters IncludeFileNameExtensions(this Azure.Search.Documents.Indexes.Models.IndexingParameters parameters, params string[] extensions) { throw null; }
```

Comment:
These are called "IndexedExtensions" on the portal.  Would be nice to have consistency.  At minimum, I prefer "IncludedFileNameExtensions" to "IncludeFileNameExtensions" unless there is huge precedent elsewhere.